examples/style_sheets/style_sheets_reference.py

```diff
@@ -98,46 +99,45 @@ def plot_histograms(ax, prng, nb_samples=10000):
     return ax
 
 
-def plot_figure(style_label=""):
+def plot_figure(axs, style_label=""):
     """Setup and plot the demonstration figure with a given style."""
     # Use a dedicated RandomState instance to draw the same "random" values
     # across the different figures.
     prng = np.random.RandomState(96917002)
-
-    # Tweak the figure size to be better suited for a row of numerous plots:
-    # double the width and halve the height. NB: use relative changes because
-    # some styles may have a figure size different from the default one.
-    (fig_width, fig_height) = plt.rcParams['figure.figsize']
-    fig_size = [fig_width * 2, fig_height / 2]
-
-    fig, axs = plt.subplots(ncols=6, nrows=1, num=style_label,
-                            figsize=fig_size, squeeze=True)
-    axs[0].set_ylabel(style_label)
+    axs[0].figure.set_facecolor(plt.rcParams['figure.facecolor'])
+    axs[0].set_ylabel('ylabel')
 
     plot_scatter(axs[0], prng)
     plot_image_and_patch(axs[1], prng)
     plot_bar_graphs(axs[2], prng)
     plot_colored_circles(axs[3], prng)
     plot_colored_sinusoidal_lines(axs[4])
     plot_histograms(axs[5], prng)
-
-    fig.tight_layout()
-
-    return fig
-
+    col = np.array([19, 6, 84])/256
+    back = mcolors.rgb_to_hsv(
+        mcolors.to_rgb(plt.rcParams['figure.facecolor']))[2]
+    if back < 0.5:
+        col = [0.8, 0.8, 1]
+    axs[0].figure.suptitle(style_label, x=0.03, fontsize=12,
+                           ha='left', color=col)
```

Comment:
This seemingly obscure workaround needs a comment as well.

Reply:
Yes fair.  I don't really consider this example a code example so wasn't too worried about being didactic.  Also this suptitle is not meant to illustrate the styles - it was meant to be distinct.